oceanprotocol/ocean.py
# Add side staking cap tests

Signed-off-by: Pablo Maldonado <pablo@oceanprotocol.com>

Fixes #753.

Changes proposed in this PR:

- Test the cap limits of the Side Staking bot
- Try to steal base token from the pool unsuccessful
- Check the "constant" ratio-> To be discussed, see my TODO
